tests/src/test/scala/common/Wsk.scala

```diff
@@ -1003,11 +1003,12 @@ trait RunWskCmd extends Matchers {
             verbose: Boolean = false,
             env: Map[String, String] = Map("WSK_CONFIG_FILE" -> ""),
             workingDir: File = new File("."),
+            stdinFile: File = null,
```

Comment:
another scala best practice recommendation to remove `null` from all scala code... removing `null` per @markusthoemmes suggestion :+1: 
http://alvinalexander.com/scala/scala-null-values-option-uninitialized-variables